<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="prikab" name="SEGMENTS ADD TESTCASES" tests="8" failures="0" timestamp="9 Nov 2016 02:25:19 GMT" time="461.884" errors="0">
  <testcase name="@BeforeMethod beforeTest" time="0.0" classname="configuration.SegmentsEdit_testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Utility.TestBase.beforeTest(TestBase.java:87)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeTest -->
  <testcase name="@BeforeMethod beforeTest" time="0.0" classname="configuration.SegmentsRestore_testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Utility.TestBase.beforeTest(TestBase.java:87)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeTest -->
  <testcase name="@AfterMethod AfterLogin" time="0.0" classname="configuration.SegmentsEdit_testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Utility.TestBase.AfterLogin(TestBase.java:105)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod AfterLogin -->
  <testcase name="@AfterMethod AfterLogin" time="0.0" classname="configuration.SegmentsRestore_testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Utility.TestBase.AfterLogin(TestBase.java:105)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod AfterLogin -->
  <testcase name="create_a_new_segment" time="73.034" classname="configuration.Segments_TestCases"/>
  <testcase name="select_cancel_button_in_segements_add_page" time="38.818" classname="configuration.Segments_TestCases"/>
  <testcase name="create_a_new_segment_without_a_clienname" time="51.733" classname="configuration.Segments_TestCases"/>
  <testcase name="create_a_new_segment_without_a_segment_group" time="54.89" classname="configuration.Segments_TestCases"/>
  <testcase name="create_a_new_segment_without_a_name" time="55.16" classname="configuration.Segments_TestCases"/>
  <testcase name="create_a_new_segment_without_a_code" time="54.614" classname="configuration.Segments_TestCases"/>
  <testcase name="create_a_new_segment_without_a_non_numerical_code" time="58.407" classname="configuration.Segments_TestCases"/>
  <testcase name="database_delete_segment_groups" time="0.037" classname="configuration.Segments_TestCases"/>
</testsuite> <!-- SEGMENTS ADD TESTCASES -->
